src/pages/settings/wallet-settings/wallet-settings-advanced/wallet-export/wallet-export.scss

```diff
@@ -7,9 +7,23 @@ page-wallet-export {
       border: 1px solid #e9e9e9;
       border-radius: 4px;
     }
+    .on-print-show {
+      display: none;
+    }
   }
   .not-supported-message {
     margin-top: 5rem;
     text-align: center;
   }
+  @media print {
```

Comment:
Try to center the image and text when print in mobile

Reply:
The CSS is only available when printing on desktop. Within cordova, only the div element <div id="cordovaPrintWrapper"> is passed to the printer. If I'm going to pass the whole <html> or <body>, the printing preview on the device is showing an empty page.